cellengine/resources/fcs_file.py

```diff
@@ -199,9 +199,14 @@ def get_events(self, inplace: bool = False, **kwargs):
 
         Returns: A DataFrame of this files data, with query parameters applied.
             If inplace=True, it updates the self.events property.
+            If destination=True, saves the file to 'destination' and returns None.
```

Comment:
```suggestion
            If destination is a string, saves the file to 'destination' and returns None.
```

`open()` probably does adequate validation that `destination` is a string, so probably don't need to change the fn body.